kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousTester.java

```diff
@@ -209,6 +213,73 @@ public interface KroxyliciousTester extends Closeable {
      */
     KafkaClient simpleTestClient(String virtualCluster);
 
+    /**
+     * Creates the given topics on the specified virtual cluster, with an admin client using the specified configuration.
+     * @param topics the topics to be created
+     * @param additionalConfig additional admin config
+     * @param virtualCluster the virtual cluster we want the topics created on
+     * @return CreateTopicsResult
+     */
+    CreateTopicsResult createTopics(String virtualCluster, Map<String, Object> additionalConfig, NewTopic... topics);
```

Comment:
hmmmm. I wonder if the tester should be offering a `TopicManager` type interface as just adding topic management  is a massive jump in the API surface of the Tester.

Reply:
+1 for a TopicManager, could also consider a pattern like `tester.topicManager("virtualCluster1").create(xyz)`.